perllib/FixMyStreet/Cobrand/Hackney.pm

```diff
@@ -143,6 +143,14 @@ around open311_extra_data_include => sub {
         $contact->email($code) if $code;
     }
 
+    if ($row->geocode) {
+        my $address = $row->geocode->{resourceSets}->[0]->{resources}->[0]->{address};
+        push @$open311_only, (
+            { name => 'closest_address', value => $address->{formattedAddress} }
+        );
+        $h->{closest_address} = '';
```

Comment:
Ah, clever. (CheshireEast/Oxfordshire/Rutland do not do this, wonder if they should? Or if it's working anyway somehow.)